src/store/wizardSlice.ts

```diff
@@ -40,6 +42,13 @@ type wizardState = {
     shareMethod: AwsShareMethod;
     source: V1ListSourceResponseItem | undefined;
   };
+  azure: {
+    tenantId: string;
+    shareMethod: AzureShareMethod;
+    subscriptionId: string;
+    source: AzureUploadRequestOptions['source_id'] | undefined;
+    resourceGroup: AzureUploadRequestOptions['resource_group'];
+  };
```

Comment:
This is very nit picky, but can we please move the shareMethod either to the beginning or to the end of the list? So it's not mixed up with the Azure account info? :sweat_smile: 

```suggestion
  azure: {
    shareMethod: AzureShareMethod;
    tenantId: string;
    subscriptionId: string;
    source: AzureUploadRequestOptions['source_id'] | undefined;
    resourceGroup: AzureUploadRequestOptions['resource_group'];
  };
```